portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,34 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> list;
+  
+  @Override
+  public void init(){
+      list = new ArrayList<>();
+      list.add("Pineapple on pizza is pretty good");
+      list.add("Dogs are overrated");
+      list.add("Alpacas are awesome");
```

Comment:
This `ArrayList` seems to be the same every time. Right now, you're recreating it every time `doGet` is called. Maybe move it to a default constructor to make sure it's only initialized once?